src/poetry/utils/shell.py

```diff
@@ -70,7 +70,13 @@ def get(cls) -> Shell:
 
     def activate(self, env: VirtualEnv) -> int | None:
         activate_script = self._get_activate_script()
-        bin_dir = "Scripts" if WINDOWS else "bin"
+        if WINDOWS:
+            bin_path = os.path.join(env.path, "Scripts")
```

Comment:
Use `pathlib` instead of `os.path`, please.

Reply:
I've made some changes, please review.